packages/vscode-extension/src/devices/AndroidEmulatorDevice.ts

```diff
@@ -480,7 +481,7 @@ export class AndroidEmulatorDevice extends DeviceBase {
 }
 
 export async function createEmulator(
-  displayName: string,
+  modelName: string,
```

Comment:
so it appears we are effectively renaming `displayName` to `deviceName` ?? If so I don't think I like that. `deviceName` suggest this is something that's associated with the device itself, while `displayName` makes it pretty obvious this can be user defined and doesn't have any hidden meaning